backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/invoicecandidate/C_OrderLine_Handler.java

```diff
@@ -264,6 +264,9 @@ private I_C_Invoice_Candidate createCandidateForOrderLine(final I_C_OrderLine or
 		icRecord.setIsNotShowOriginCountry(order.isNotShowOriginCountry());
 		icRecord.setC_PaymentInstruction_ID(order.getC_PaymentInstruction_ID());
 
+		setHarvestingDetails(icRecord, order);
```

Comment:
here we set the harvesting details for a new IC. but can't onesomehow change the order's harvesting details after the ICs was created; e.g. reactivate the order and change the harvesting-details?

Reply:
that's right. code changes added